src/layouts/DialogLayout/DialogLayout.stories.tsx

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Dialog } from '../../components/containers/Dialog/Dialog.tsx';
+import { Logo } from '../../layouts/AppLayout/Logo/Logo.tsx';
+import { H3 } from '../../typography/Heading/Heading.tsx';
+import { Form } from '../../components/forms/context/Form/Form.tsx';
+import { FormLayout } from '../FormLayout/FormLayout.tsx';
+import { CheckboxGroup } from '../../components/forms/controls/CheckboxGroup/CheckboxGroup.tsx';
+import { InputField } from '../../components/forms/fields/InputField/InputField.tsx';
+import { Button } from '../../components/actions/Button/Button.tsx';
+import { Stepper } from '../../components/navigations/Stepper/Stepper.tsx';
+import { Icon } from '../../components/graphics/Icon/Icon.tsx';
+import { FieldLayout } from '../../components/forms/fields/FieldLayout/FieldLayout.tsx';
+
+import { DialogLayout } from './DialogLayout';
+
+type DialogLayoutArgs = React.ComponentProps<typeof DialogLayout>;
+type Story = StoryObj<DialogLayoutArgs>;
+
+export default {
+  tags: ['autodocs'],
+  component: DialogLayout,
+  parameters: {
+    layout: 'padded',
+  },
+} satisfies Meta<DialogLayoutArgs>;
+
+
+export const DialogOnboarding: Story = {
+  render: () => {
+    return (
+      <Dialog
+        // TODO: Replace here after https://github.com/fortanix/baklava/issues/163
+        title={<Logo subtitle="Armor"/>}
+        showCancelAction={false}
+        onRequestClose={() => console.log('close')}
+      >
+        <DialogLayout
+          title="Let's Connect to Your Cloud Provider"
+          aside={(
+            <>
+              <Stepper
+                steps={[
+                  { stepKey: '1', title: 'Select Cloud Provider' },
+                  { stepKey: '2', title: 'Setup Cloud Connection' },
+                  { stepKey: '3', title: 'Set Up Subscriptions' },
+                  { stepKey: '4', title: '4th Option Title' },
+                ]}
+                activeKey="1"
+                onSwitch={() => {}}
+              />
+              {/* TODO: Maybe this should be a separate component? */}
+              <hr/>
+              <div style={{display: 'flex', flexDirection: 'row', gap: '4px', alignItems: 'flex-start'}}>
+                <p><Icon icon="info"/></p>
+                <p><small>Step-by-Step Guide to Connect to Azure Cloud Provider</small></p>
```

Comment:
Let's not use the `<small>` element, since it doesn't really add anything in terms of semantics.